tutorial/get-started-with-openstack.rst

```diff
@@ -20,9 +20,10 @@ You will only need one dedicated physical machine with:
 
 * 4+ core amd64 processor
 * minimum of 16 GiB of RAM
-* minimum of 100 GiB SSD storage on the ``rootfs`` partition
+* minimum of 100 GiB SSD storage on the ``rootfs`` partition (more may be required if additional features are enabled that require persistent storage)
```

Comment:
```suggestion
* minimum of 100 GiB SSD storage on the ``rootfs`` partition
```

The get-started goal does not explore features, and asks for a minimum of `100`GiB, we might expand the sizing section in architecture to take into account features?